lib/features/map_view/widgets/map_atrribution.dart

```diff
@@ -0,0 +1,22 @@
+import "package:flutter/material.dart";
+import "package:flutter_map/flutter_map.dart";
+import "package:flutter_riverpod/flutter_riverpod.dart";
+
+import "../../../utils/launch_url_util.dart";
+
+/// legal requirements for using OpenStreetMap
+class OpenMapAtrribution extends ConsumerWidget {
+  const OpenMapAtrribution({super.key});
+
+  @override
+  Widget build(BuildContext context, WidgetRef ref) {
+    return RichAttributionWidget(
+      attributions: [
+        TextSourceAttribution(
+          "OpenStreetMap contributors",
+          onTap: () async => ref.launch("https://openstreetmap.org/copyright"),
```

Comment:
Maybe an overkill but what about extracting that url?

Reply:
cool, no problem